website/_kb/user-guide/querying/favorites-tagging.md

```diff
@@ -0,0 +1,22 @@
+---
+category: querying
+parent_category: user-guide
+helpscout_url: https://help.redash.io/article/45-query-filters
+title: Favorites & Tagging
+slug: favorites-tagging
+---
+
+Redash users write a lot of queries and dashboards! Favorites and Tagging are here to make finding them easy as your list of queries grows from a few hundred to a few thousand.
+
+<img src="/assets/images/docs/gitbook/favorites-example.png" width="100%">
+
+## Favorites
+You can favorite a dashboard or query by clicking the star to the left of its title anywhere in Redash. The star will turn gold to indicate success. Your list of favorites of each kind can be viewed from the query or dashboard list view. Just click the `Favorites` filter on the right.
```

Comment:
It's yellow, not gold. :)

Reply:
We should mention that favorites are also available from the homepage and navbar dropdowns.